core/vm_env.go

```diff
@@ -41,30 +41,26 @@ func GetHashFn(ref common.Hash, chain *BlockChain) func(n uint64) common.Hash {
 }
 
 type VMEnv struct {
-	state *state.StateDB // State to use for executing
-	evm   *vm.EVM        // The Ethereum Virtual Machine
-	depth int            // Current execution depth
-	msg   Message        // Message appliod
+	chainConfig *ChainConfig   // Chain configuration
```

Comment:
Will keep so that we can do `env.chainConfig` instead of `env.blockchain.Config()`.